src/main/java/org/spongepowered/common/config/category/WorldCategory.java

```diff
@@ -77,6 +77,9 @@
                                                     + "then we recommend disabling it.")
     private boolean denyChunkRequests = false;
 
+    @Setting(value = "deny-te-chunk-requests", comment = "If 'true', TileEntities are not allowed to load chunks when calling getBlockState.")
```

Comment:
This needs to be mapped to tile entity id's because certain tile entities do need to be allowed to load chunks (otherwise you break some modded tile entities that depend on random stuff)